src/CSSList/Document.php

```diff
@@ -109,6 +109,26 @@ public function getSelectorsBySpecificity($sSpecificitySearch = null)
         return $aResult;
     }
 
+    /**
+     * Returns all `CSSFunction` objects recursively found in the tree, no matter how deeply nested the rule sets are.
+     * 
+     * @param CSSList|RuleSet|string $mElement
+     *        the `CSSList` or `RuleSet` to start the search from (defaults to the whole document).
+     *        If a string is given, it is used as rule name filter.
+     *
+     * @return array<int, CSSFunction>
+     */
+    public function getAllFunctions($mElement = null)
```

Comment:
Nitpick: Could we think of a better name for this? Maybe `getFunctionsRecursively()`?

Reply:
> Nitpick: Could we think of a better name for this? Maybe `getFunctionsRecursively()`?

I'm following the naming scheme of the file eg. getAllValues, getAllRulesets, etc.